controllers/volumereplicationgroup_controller.go

```diff
@@ -121,13 +122,16 @@ func (r *VolumeReplicationGroupReconciler) pvcMapFunc(obj client.Object) []recon
 }
 
 func (r *VolumeReplicationGroupReconciler) configMapFun(configmap client.Object) []reconcile.Request {
-	log := ctrl.Log.WithName("configmap").WithName("VolumeReplicationGroup")
```

Comment:
These logs are useful to know when a event triggered a reconcile. Although not consistent in all event handlers, I would still prefer to retain these.